doc/whats_new.rst

```diff
@@ -22,6 +22,7 @@ Current
 Changelog
 ~~~~~~~~~
 
+- Added handedness to participant files, by `Dominik Welke`_ (`#354 <https://github.com/mne-tools/mne-bids/pull/354>`_)
```

Comment:
you should add the change to the bottom of the changelog not the top.

Reply:
ok, next time ill do that..